### PR TITLE
[feature-wip](workload-group) Support setting user default workload group

### PR DESCRIPTION
**Pull request by luozenglin**
## Proposed changes

Issue Number: close #xxx

 ```
SET PROPERTY 'default_workload_group' = 'group_name';
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luozenglin** [comment]
run buildall

**Comment by luozenglin** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java [hunk 1]
```diff
@@ -69,6 +69,8 @@ public class UserProperty implements Writable {
     private static final String PROP_QUOTA = "quota";
     private static final String PROP_DEFAULT_LOAD_CLUSTER = "default_load_cluster";
 
+    private static final String PROP_WORKLOAD_GROUP = "workload_group";
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java:72]: using default_workload_group

[the remark this answers is not in the document]
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java]: done

File: fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java [hunk 2]
```diff
@@ -174,6 +181,7 @@ public void update(List<Pair<String, String>> properties) throws UserException {
         long execMemLimit = this.commonProperties.getExecMemLimit();
         int queryTimeout = this.commonProperties.getQueryTimeout();
         int insertTimeout = this.commonProperties.getInsertTimeout();
+        String workloadGroup = this.commonProperties.getWorkloadGroup();
```
[the remark this answers is not in the document]
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java:184]: done

File: fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java (hunk 1 above)
Comment (luozenglin) [reply fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java]: done

**Comment by luozenglin** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**